django_cloud_tasks/tasks/task.py

```diff
@@ -385,6 +394,30 @@ def eager(cls) -> bool:
     def _get_tasks_client(cls) -> CloudTasks:
         return CloudTasks()
 
+    @classmethod
+    def enqueue_retry_policy(cls) -> retry.Retry | None:
+        def import_exceptions_class() -> None:
+            exceptions_classes = []
+            for exception in enqueue_retry_exceptions:
+                module_name, class_name = exception.rsplit(".", 1)
+                module = __import__(module_name, fromlist=[class_name])
+                exception_class = getattr(module, class_name)
+                exceptions_classes.append(exception_class)
+            return exceptions_classes
+
+        enqueue_retry_exceptions = cls.enqueue_retry_exceptions or get_config(name="enqueue_retry_exceptions")
+        if not enqueue_retry_exceptions:
+            return
+
+        exceptions_classes = import_exceptions_class()
+        return retry.Retry(
+            predicate=retry.if_exception_type(*exceptions_classes),
+            initial=cls.enqueue_retry_initial or get_config(name="enqueue_retry_initial"),
+            maximum=cls.enqueue_retry_maximum or get_config(name="enqueue_retry_maximum"),
+            multiplier=cls.enqueue_retry_multiplier or get_config(name="enqueue_retry_multiplier"),
+            deadline=cls.enqueue_retry_deadline or get_config(name="enqueue_retry_deadline"),
```

Comment:
I don't know if it's worth testing, but will it work if I set just one of these and the others are `None`? Will the default work if we set it as None?

Reply:
Nice catch!
I've change the implementation to utilize Google's built-in defaults for all retry parameters (initial, maximum, multiplier, deadline), if no value is explicitly set in the app configuration.
The exception list (enqueue_retry_exceptions) remains the only parameter with a default of None - this ensures the retry mechanism is strictly opt-in and only activates when explicitly configured.